Update 8/6/18

wheikoff opened the pull request.

kbuzard commented on Will Heikoff Documentation:
```diff
@@ -321,3 +321,14 @@ The assumption that the Zip Code Business Pattern Data is based off of USPS Zip
+	ZBP Manufa. Data has 0 Zips with no Manufa. Employment
+Earliest ZCTA Shapefiles available are 2007
+Downloaded 2000 ZCTA Cartographic Boundary Files for California ftp://ftp2.census.gov/geo/tiger/PREVGENZ/zt/z500shp/, titled 
+	zt06_d00.shp, saved in G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Complete Zip Code Industry Detail File
```
What's the difference between a shapefile and a cartographic boundary file?

wheikoff commented:
From my understanding its a simplified version of a ZCTA shapefile (not as detailed). The full description is at https://www.census.gov/geo/maps-data/data/cbf/cbf_description.html. I'm still looking if I can find a ZCTA shapefile somewhere.

some
Cartographic Boundary File Description - Geography - U.S ...<https://www.census.gov/geo/maps-data/data/cbf/cbf_description.html>
www.census.gov
Our statistics highlight trends in household and family composition, describe characteristics of the residents of housing units, and show how they are related.


________________________________
From: Kristy Buzard <notifications@github.com>
Sent: Monday, August 6, 2018 2:58:08 PM
To: kbuzard/labs
Cc: William Heikoff; Author
Subject: Re: [kbuzard/labs] Update 8/6/18 (#16)


@kbuzard commented on this pull request.

________________________________

In Will Heikoff Documentation<https://github.com/kbuzard/labs/pull/16#discussion_r208027219>:

> @@ -321,3 +321,14 @@ The assumption that the Zip Code Business Pattern Data is based off of USPS Zip
        discretion of the U.S. Postal Service and may change from time to time”. This information can be found at the census business
        patterns documentation website at https://www.census.gov/programs-surveys/cbp/technical-documentation/methodology.html

+
+8/6/18
+Adjusted CA_WAC_Transformation.R file to format the Block ID to add the necessary 0 at the beginning of the ID directly in R rather
+       than having to do it in ArcMap
+Updated ZIPs_Tot_Emp.R file to extract only California Zip Codes from the file
+2000 ZCTA Manufa. Data has 198 Zips with no Manufa. Employment, 2000 ZBP Manufa. Data has 0 Zips with no Manufa. Employment, 1998
+       ZBP Manufa. Data has 0 Zips with no Manufa. Employment
+Earliest ZCTA Shapefiles available are 2007
+Downloaded 2000 ZCTA Cartographic Boundary Files for California ftp://ftp2.census.gov/geo/tiger/PREVGENZ/zt/z500shp/, titled
+       zt06_d00.shp, saved in G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Complete Zip Code Industry Detail File


What's the difference between a shapefile and a cartographic boundary file?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/kbuzard/labs/pull/16#pullrequestreview-143758032>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AmkU6tmV2I7R7g92CdrXHoItuRDaHrwWks5uOK3ggaJpZM4Vw_70>.
